e2e-tests/test_client.py

```diff
@@ -1,11 +1,15 @@
+from gevent import monkey, Timeout
```

Comment:
Can we remove the depenency on gevent?

Reply:
I was including it to show how someone can set a custom time limit on `client.wait`. Removed it in favor of eventually adding a `timeout` parameter to `client.wait`.